test/jdk/sun/tools/jstatd/JstatdTest.java

```diff
@@ -350,7 +350,10 @@ private void runTest(boolean useShortSyntax) throws Throwable {
 
         // Verify output from jstatd
         OutputAnalyzer output = jstatdThread.getOutput();
-        output.shouldBeEmptyIgnoreVMWarnings();
+        List<String> stdout = output.asLinesWithoutVMWarnings();
+        output.reportDiagnosticSummary();
+        assertEquals(stdout.size(), 1, "Output should contain one line");
+        assertTrue(stdout.get(0).startsWith("jstatd started"), "List should start with 'jstatd started'");
         assertNotEquals(output.getExitValue(), 0,
```

Comment:
Before your fix, was the "jstatd started" line being missed because of this bug.